packages/perseus-score/src/empty-diner-check.test.ts

```diff
@@ -0,0 +1,288 @@
+import emptyDinerCheck from "./empty-diner-check";
+
+import type {UserInputMap} from "./validation.types";
+import type {PerseusItem, PerseusRenderer} from "@khanacademy/perseus-core";
+
+function generatePerseusItem(question: PerseusRenderer): PerseusItem {
```

Comment:
Non-blocking, but would you mind making a TODO to come back and merge this with the one in Perseus?  Or maybe it's not that big a deal to have two similar test helpers in different packages. 

https://github.com/Khan/perseus/blob/c8e383b469426182a1392ca6ad2cde21b61e2f40/packages/perseus/src/util/test-utils.ts#L61

Reply:
I'll file a ticket. I just couldn't import from `perseus` into `perseus-score`. I think we just need a central place to help construct objects.